### DISCUSSION TITLE
how necessary exactly is inetutils other than hostname?

### DISCUSSION THREAD

**Discussion by Tcll** [OpenRC/openrc#618] [Q&A]
After overviewing my packages in Artix, I noticed there wasn't anything I needed inetutils for aside from openrc

after looking into it a bit, I'd noticed openrc uses the hostname command, which I was able to replace with:
`echo "$h" > /proc/sys/kernel/hostname`

I'm wondering if there's anything else inetutils is used for within openrc
or if it's now safe to -Rdd

**Comment by vapier** [comment]
what exactly are you defining as "inetutils" ?  i have no idea what "Artix" is.

**Comment by Tcll** [comment]
here's the binary files provided by the inetutils-2.3-1 package on Artix

```
usr/bin/dnsdomainname
usr/bin/ftp
usr/bin/ftpd
usr/bin/hostname
usr/bin/rcp
usr/bin/rlogin
usr/bin/rlogind
usr/bin/rsh
usr/bin/rshd
usr/bin/talk
usr/bin/talkd
usr/bin/telnet
usr/bin/telnetd
```

this is used by `/etc/init.d/hostname`, which I've removed in my modification above
I'm only wondering if OpenRC makes use of anything else here, since it's a dependency listed here:
https://packages.artixlinux.org/details/openrc

>  i have no idea what "Artix" is.

Artix is just an arch-based distro that makes use of OpenRC :)
the repo linked above is also used by that distro

**Comment by vapier** [comment]
OpenRC doesn't use any of those programs

**Comment by Tcll** [comment]
alright thanks, that helps a lot :)
guess it's all just the work of whatever the Artix devs do in init.d/ then
